src/core/segment_allocator.cc

```diff
@@ -14,8 +14,8 @@ SegmentAllocator::SegmentAllocator(mi_heap_t* heap) : heap_(heap) {
   constexpr size_t limit = 1ULL << 35;
   static_assert((1ULL << (kSegmentIdBits + kSegmentShift)) == limit);
   // mimalloc uses 32MiB segments and we might need change this code if it changes.
-  static_assert(kSegmentShift == MI_SEGMENT_SHIFT);
-  static_assert((~kSegmentAlignMask) == (MI_SEGMENT_MASK));
+  //  static_assert(kSegmentShift == MI_SEGMENT_SHIFT);
```

Comment:
we pulled new version of mimalloc
if this assert fails you should clean up your mimalloc build